server/src/main/kotlin/ir/armor/tachidesk/model/dataclass/MangaDataClass.kt

```diff
@@ -1,14 +1,14 @@
 package ir.armor.tachidesk.model.dataclass
 
+import ir.armor.tachidesk.model.database.table.MangaStatus
```

Comment:
import should be after copyright notice

```
<package declaration>
<empty line>
<copyright notice>
<empty line>
<imports>
```

Reply:
Should probably resolve this by using the intellij copywrite option eventually, manually moved the import